source/docs/articles/sites/files.md

```diff
@@ -1 +1,12 @@
-draft: true
+---
+title: Pantheon Filesystem
+description: Learn how to work with the Pantheon filesystem.
+keywords: code, commit, sftp, files, filesystem, file location, directory location, directory, codebase
+---
+The Pantheon architecture is comprised of highly available [application containers](/docs/articles/architecture/all-about-application-containers/) that are seamlessly integrated with Valhalla, our cloud-based filesystem.
```

Comment:
Let's lead with the platform definition of "files".  - the files directory, how to access via SFTP connection info, `cd files` or in client. It's important to note they're not local to the application container, are gitignored.